projects/flow.groups-2D-random-walk/src/project.py

```diff
@@ -0,0 +1,211 @@
+"""Define the operations for the 2D Gaussian Random Walk project."""
+
+import flow
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# Helper functions
+def get_replica(jobs, replica):
+    """Get a certain replica from a group of jobs."""
+    try:
+        job = next(filter(lambda job: job.sp.replica == replica, jobs))
+    except StopIteration:
+        raise RuntimeError(f"No replica {replica} found.")
+    return job
+
+
+def generate_stores(jobs, store_name):
+    """Yield a data store for each job in jobs."""
+    for job in jobs:
+        with job.data:
+            yield job.data[store_name]
+
+
+class RandomWalkProject(flow.FlowProject):
+    """Create a workflow for simulating 2D Gaussian random walks."""
+
+    pass
+
+
+@RandomWalkProject.label
+def simulated(job):
+    """Return whether the job simulated."""
+    return "positions" in job.data
+
+
+def all_simulated(*jobs):
+    """Return whether all jobs simulated."""
+    return all(simulated(job) for job in jobs)
+
+
+# Operations on individual jobs
+
+# Group for operations on single jobs that are reachable from simulate
+base = RandomWalkProject.make_group("base")
+
+
+@base
+@RandomWalkProject.post(simulated)
+@RandomWalkProject.operation
+def simulate(job):
+    """Simulate a 2D random walk."""
+    # use job id as random seed
+    n_steps = job.doc.run_steps
+    generator = np.random.default_rng(int(job.id, 16))
+    # Generate all moves at once since they are independent
+    moves = generator.normal(job.sp.mean, job.sp.standard_deviation, (n_steps, 2))
+    position = np.zeros((n_steps + 1, 2), dtype=float)
+    # Store the results: cumsum aggregates all previous moves from the origin
```

Comment:
Avoid the use of the word "aggregates" here because it's potentially confusing.
```suggestion
    # Store the results: perform a cumulative sum of all moves starting from the origin
```